diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -118,17 +118,21 @@ fun ASTNode.isNodeFromCompanionObject(): Boolean {
     return false
 }
 
+fun ASTNode.isConstant(): Boolean {
+    return (this.isNodeFromCompanionObject() || this.isNodeFromFileLevel()) && this.isValProperty() && this.isConst()
```

Comment:
What about #70? How does it affect the logic here? I think, `isConstant` should return true for all objects, which `isNodeFromCompanionObject` currently checks, but it seems to be wrong behavior.

Reply:
yes, it affects. I have found several bugs even in tests. Fixed them in https://github.com/cqfn/diKTat/pull/100

Reply:
I mean, if `isNodeFromCompanionObject` will be corrected, does this code need to be changed so that constants from other objects are detected too?

Reply:
good point, it is missing in code style

Reply:
fixed